src/components/Calculator.js

```diff
@@ -3,37 +3,158 @@ import React, { Component } from 'react';
 // eslint-disable-next-line react/prefer-stateless-function
 class Calculator extends Component {
   render() {
+    const {
+      // eslint-disable-next-line react/prop-types
+      total, operation, next, onBtnClick,
+    } = this.props;
     return (
       <div className="container">
-        <div className="result-container">0</div>
+        <div className="result-container">
+          <span>{total}</span>
+          <span>{operation}</span>
+          <span>{next}</span>
+        </div>
         <div className="first-row">
-          <button type="button">AC</button>
-          <button type="button">+/-</button>
-          <button type="button">%</button>
-          <button type="button">÷</button>
+          <button type="button" onClick={onBtnClick} id="AC">
+            AC
+          </button>
+          <button type="button" onClick={onBtnClick} id="+/-">
+            +/-
+          </button>
+          <button type="button" onClick={onBtnClick} id="%">
+            %
+          </button>
+          <button type="button" onClick={onBtnClick} id="÷">
+            ÷
+          </button>
         </div>
         <div className="second-row">
-          <button type="button" className="col">7</button>
-          <button type="button" className="col">8</button>
-          <button type="button" className="col">9</button>
-          <button type="button" className="col">*</button>
+          <button
+            type="button"
+            className="col"
+            onClick={onBtnClick}
+            id="7"
+          >
+            7
+          </button>
+          <button
+            type="button"
+            className="col"
+            onClick={onBtnClick}
+            id="8"
+          >
+            8
+          </button>
+          <button
+            type="button"
+            className="col"
+            onClick={onBtnClick}
+            id="9"
+          >
+            9
+          </button>
+          <button
+            type="button"
+            className="col"
+            onClick={onBtnClick}
+            id="*"
+          >
+            *
+          </button>
```

Comment:
- [ ] Nice implementation of the different logical operations of the app but your app generates an error when I try to multiply to different numbers and click the equal sign to get the result. Please try to fix this
![multiply](https://user-images.githubusercontent.com/55596970/171502059-47a9af8a-9839-457c-b71e-873e29357804.png)